src/components/Share/Share.tsx

```diff
@@ -1,25 +1,28 @@
 import { FC } from 'react'
 import Link from 'next/link'
+import { Address } from 'wagmi'
 
 import { TwitterIcon } from '@/components/icons/Twitter'
 import { useMintDialogContext } from '@/components/MintDialog/Context/useMintDialogContext'
+import { website } from '@/config/website'
 import { WarpCast } from '@/components/icons/Warpcast'
 import { Lens } from '@/components/icons/Lens'
+import { useEns } from '@/utils/useEns'
 
 const twitterURL: string = 'https://twitter.com/intent/tweet'
 const warpCastURL: string = 'https://warpcast.com'
 const lensURL: string = 'https://lenster.xyz'
 
 type ShareComponentProps = {}
 export const Share: FC<ShareComponentProps> = () => {
-  const { dropName } = useMintDialogContext()
-  const {
-    location: { href },
-  } = window
+  const { dropName, partnerName, creatorAddress } = useMintDialogContext()
+  const { name } = useEns(creatorAddress as Address)
+
+  const href = `${website.url}/${partnerName}`
```

Comment:
I think won't be the right URL sometimes. E.g. Coca Cola and partners with spaces in their names will have a separate slug. Can we look into getting the slug from the config and using that instead?

Reply:
CC @arpitsrivastava-cb @lauchness @DjordjeVujatovic @ximxim

Reply:
@AshishYelekar added new PR https://github.com/base-org/onchainsummer.xyz/pull/204 since I am unable to change in this PR required lot of rebase.Please have a look